System.IO.Ports/SerialPort.cs

```diff
@@ -51,6 +53,15 @@ public sealed class SerialPort : IDisposable
         private string _newLine;
         private int _bufferSize = 256;
 
+        /// <summary>
+        /// Gets an array of serial port names for the current computer.
+        /// </summary>
+        /// <returns>An array of serial port names for the current computer.</returns>
+        public static string[] GetPortNames()
```

Comment:
Is there any real value on having this in the API? I mean, if needed this `Split` can be used in the caller code.